code/__DEFINES/colors.dm

```diff
@@ -26,6 +26,8 @@
 #define COLOR_BLUE_LIGHT       "#33CCFF"
 #define COLOR_NAVY             "#000080"
 #define COLOR_PINK             "#FFC0CB"
+#define COLOR_BLUSH_PINK       "#DE5D83"
+#define COLOR_BLUSH_POLYSMORPH "#00FF32"
```

Comment:
```suggestion
#define COLOR_BLUSH_PINK       "#DE5D83"
#define COLOR_BLUSH_POLYSMORPH "#00FF32"
```
this feels out of place but it'd need further input tbh

you could define it on the place you use the colors instead